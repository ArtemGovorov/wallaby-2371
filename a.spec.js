it('snapshot updating should work', () => {
  expect({a: 1}).toMatchSnapshot();
});
