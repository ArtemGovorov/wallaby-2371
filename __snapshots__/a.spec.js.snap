// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot updating should work 1`] = `
Object {
  "a": 1,
}
`;
